CMakeLists.txt

```diff
@@ -43,7 +43,15 @@ function(add_subdirectory_exclude_platforms NAME)
 endfunction()
 
 # Add blink example
-add_subdirectory_exclude_platforms(blink)
+if (PICO_CYW43_SUPPORTED) # set by PICO_BOARD=pico_w
```

Comment:
Can we make this `if (PICO_CYW43_SUPPORTED AND NOT TARGET pico_cyw43_arch)`

Reply:
That might be better